.github/workflows/release.yml

```diff
@@ -112,22 +111,10 @@ jobs:
           export GH_PAGES_DIR="$PWD/gh-pages"
           ./build/dev/publish-plugin-registry-to-gh-pages.sh
       - name: publish to npmjs
+        if: ${{ github.event.inputs.publishToNpmJs == 'true' }}
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
         run: |
+          BUILDER=docker SKIP_FORMAT=true SKIP_LINT=true SKIP_TEST=true ./build.sh --tag "${{ github.event.inputs.version}}"
           cd tools/build
           npm publish --tag latest
-      - name: Create failure MM message
-        if: ${{ failure() }}
```

Comment:
instead of removing the MM notifications, perhaps we can have different ones based on input? if a release, we want to see MM notifications! and if re-releasing npmjs stuff, we ALSO want to see if that passed/failed.